libvirt/README.md

```diff
@@ -19,6 +19,13 @@ You must have a Linux/KVM system with libvirt installed and the following tools:
 Assume that you have a 'default' network and storage pools created in libvirtd system instance (`qemu:///system`). However,
 if you have a different pool name then the scripts should be able to handle it properly.
 
+**Note:** Installation of the [kcli](https://kcli.readthedocs.io/en/latest/#dev-installation) is supported exclusively for s390x machines using pypi. Follow these simple commands for installation on one ubuntu20.04 s390x vsi:
+```bash
+apt-get update
+apt-get -y install python3-pip genisoimage qemu-kvm libvirt-daemon-system libvirt-dev cpu-checker
+pip3 install kcli
```

Comment:
@liudalibj I guess this is to workaround the problem described in https://github.com/confidential-containers/cloud-api-adaptor/issues/1593#issuecomment-1822529482, right?

Reply:
yeah, after kcli support s390x deb package, when we can use the install script directly.
eg:
```
curl https://raw.githubusercontent.com/karmab/kcli/main/install.sh | sudo bash
```